jcv-db-assertj-db/build.gradle.kts

```diff
@@ -41,11 +40,17 @@ tasks {
 
     withType<DokkaTask> {
         reportUndocumented = false
+    }    
+
+    val version: String by project
+    if (version.endsWith("-SNAPSHOT")) {
```

Comment:
Hi, I think we also need it for release.

Reply:
Hi, the metadata generation works well for release. I tried it locally.

Reply:
If I remember properly it's maven central that doesn't like it.